modules/tests/src/test/scala/lucuma/itc/service/GraphQLCustomSedSuite.scala

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.itc.service
+
+import io.circe.literal.*
+
+class GraphQLCustomSedSuite extends GraphQLSuite {
+
+  test("custom sed integration time") {
+    query(
+      """
+        query {
+          spectroscopyIntegrationTime(input: {
+            exposureTimeMode: {
+              signalToNoise: {
+                value: 2,
+                at: { nanometers: 600 }
+              }
+            },
+            asterism: [
+              {
+                sourceProfile: {
+                  point: {
+                    bandNormalized: {
+                      sed: {
+                        fluxDensities: [
```

Comment:
How big are SED? I wonder if there is a more compact format, perhaps csv or even binary